README.md

```diff
@@ -28,17 +28,18 @@ API documentation:
 
 ## Pull Requests
 
-See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md).
+Please read [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) 
+before submitting your pull requests.
```

Comment:
"Please read CONTRIBUTING.md before submitting any issues, or creating your pull requests." 

other than that, :+1:  looks good to me